plugins/hls-tactics-plugin/src/Wingman/Judgements.hs

```diff
@@ -34,6 +34,9 @@ buildHypothesis
   where
     go (occName -> occ, t)
       | Just ty <- t
+      -- TODO what's the intention behind this check?
+      -- can it be replace with something like `not . isSymOcc`?
+      -- https://hackage.haskell.org/package/ghc-9.2.1/docs/GHC-Plugins.html#v:isSymOcc
```

Comment:
Probably question for @isovector 
My intention is to get rid of this usage of `head`

Reply:
This is checking that the bindings in scope were created in source Haskell. GHC makes lots of bindings whose name start with `$` or other odd things.

You could replace it with `maybe False isAlpha $ headMay $ occNameString occ`